ops/docker/Dockerfile.packages

```diff
@@ -79,26 +79,38 @@ RUN git submodule update --init --recursive
 
 RUN pnpm build
 
+FROM base as chain-mon
+WORKDIR /opt/optimism/packages/chain-mon
+ENTRYPOINT ["pnpm", "run"]
+
+# TODO keeping the rest of these here for now because they are being used
+# but we should really delete them we only need one image
 FROM base as replica-mon
 WORKDIR /opt/optimism/packages/chain-mon
-ENTRYPOINT ["pnpm", "run", "start:replica-mon"]
+ENTRYPOINT ["pnpm", "run"]
```

Comment:
Can we set the `ENTRYPOINT` once in `base`?

Reply:
I actually have no idea if that works interested to try

Reply:
Looks like the answer is yes!